src/drivers/redis/Command.php

```diff
@@ -55,4 +56,32 @@ public function actionListen($wait = 3)
     {
         $this->queue->listen($wait);
     }
+
+    /**
+     * Clears the queue.
+     */
+    public function actionClear()
+    {
+        if ($this->confirm('Are you sure?')) {
```

Comment:
you have confirm only in redis implementation?

Reply:
No, confirmation is used with all driver that have `clear` command. File, DB and Redis dirvers.